roles/KubevirtCommonTemplatesBundle/tasks/main.yml

```diff
@@ -8,6 +8,28 @@
   with_items: "{{ lookup('file', 'common-templates-'+ version +'.yaml').split('\n---\n') | select('search', '(^|\n)[^#]') | list }}"
   register: ct_status
 
+# Get all templates (including templates deployed by a previous version)
+- name: Fetching all templates to inject ownerReference
+  set_fact:
+    templates: "{{ lookup('k8s', api_version=ct_status.results[0].result.apiVersion, kind='template', label_selector='template.kubevirt.io/type=base') }}"
+
+# Inject ownerReferences to all templates
+- name: Inject ownerRef into the Templates definition
```

Comment:
I suggest you create a new role - ClaimOwnership - and invoke it from here (and from the other places). Just to avoid repeating this block so much.

```
- include_role:
    name: ClaimOwnership
    vars:
      object: "{{ item }}"
      owner: "{{ cr_info }}" -- and you might be able to omit this one if you use cr_info everywhere consistently
```

Reply:
You can be more fancy even and create a module too. It is just a Python file..

Reply:
Will try creating a new role, thanks!

Reply:
Done